extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,473 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+import moment from 'moment';
```

Comment:
This is not a dependency in `package.json`. It works when you're symlinking and building for PWA, but it will explode if you try to use this extension as a runtime extension.

Reply:
Good catch, it was probably VSCode being funky and clever when the word moment was written somewhere. Doesn't look like this was intended to be used anyway.

Reply:
@dannyrb interesting, maybe we could add this as well? https://github.com/depcheck/depcheck#readme maybe this package can do the job?